tests/php/test-class-amp-base-sanitizer.php

```diff
@@ -6,15 +6,17 @@
  */
 
 use AmpProject\Dom\Document;
+use AmpProject\AmpWP\Tests\Helpers\PrivateAccess;
 use AmpProject\AmpWP\Tests\Helpers\StubSanitizer;
 use AmpProject\AmpWP\Tests\TestCase;
 
 /**
  * Test AMP_Base_Sanitizer_Test
  *
- * @covers AMP_Base_Sanitizer
+ * @coversDefaultClass AMP_Base_Sanitizer
```

Comment:
Just in case if this test class gets `namespace`d.

```suggestion
 * @coversDefaultClass \AMP_Base_Sanitizer
```

Reply:
Is this needed since there is no namespace in this class file?

Reply:
Oh it's technically not needed since the test class nor the class being tested has a namespace. This would prevent any code coverage warnings from occurring if either class does get a namespace.